src/test/java/com/google/api/codegen/testdata/nodejs_doc_version_index_library.baseline

```diff
@@ -44,28 +44,33 @@ const VERSION = require('../package.json').version;
  * Service comment may include special characters: <>&"`'@.
  *
  * @param {object=} options - [Configuration object](#/docs).
- * @param {string=} options.key - API key to use form communication with the
- *     service.
- * @param {string=} options.keyFilename - Path to a key file to use for an
- *     API key.
- * @param {(string|number)=} options.logLevel - An integer from 0-5
- *     inclusive or a decimal representation of an integer from 0-5 inclusive
- *     in String form.
+ * @param {boolean} [options.autoRetry=true] - Automatically retry requests
+ *     if the response is related to rate limits or certain intermittent
+ *     server errors. We will exponentially backoff subsequent requests by
+ *     default.
+ * @param {object=} options.credentials - Credentials object.
+ * @param {string=} options.credentials.client_email
+ * @param {string=} options.credentials.private_key
+ * @param {string=} options.email - Account email address. Required when using a
+ *     .pem or .p12 keyFilename.
+ * @param {string=} options.keyFilename - Full path to the a .json, .pem, or
+ *     .p12 key downloaded from the Google Developers Console. If you provide
+ *     a path to a JSON file, the projectId option above is not necessary.
+ *     NOTE: .pem and .p12 require you to specify options.email as well.
+ * @param {number} [options.maxRetries=3] - Maximum number of automatic
```

Comment:
@landrito is correct. There is a parameter for controlling total retry time which should be documented instead, because it's consistent with how we configure automated retries in every other language.

This is a hidden parameter for use by our manual layers only, until @vam-google can setup an experiment to objectively determine how user prefer to configure their automatic retries.